contracts/governance/Governor.sol

```diff
@@ -109,7 +109,6 @@ abstract contract Governor is Context, ERC165, EIP712, IGovernor, IERC721Receive
                 this.getVotesWithParams.selector) ||
             // Previous interface for backwards compatibility
             interfaceId == (type(IGovernor).interfaceId ^ type(IERC6372).interfaceId ^ this.cancel.selector) ||
-            interfaceId == type(IGovernor).interfaceId ||
```

Comment:
Why are we removing this?

Reply:
Yes, also I'm not sure we want to have that many ERC165 identifier that we'll have to maintain forever.